### PR TITLE
Have a working example of mounting gcsfuse for web tasks

### PR DESCRIPTION
**Pull request by sirdarckcat**

### CODE REVIEW

File: dist/challenge-templates/web/challenge/Dockerfile [hunk 1]
```diff
@@ -29,35 +29,44 @@ RUN curl -sSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add
...
+RUN ln -s /etc/apache2/mods-available/actions.load /etc/apache2/mods-enabled/actions.load
+RUN ln -s /chroot/web-apps /web-apps
+COPY cgi-bin /usr/lib/cgi-bin
+
+COPY apache2-kctf-nsjail.conf /etc/apache2/conf-enabled/
+
+COPY web-servers.nsjail.cfg /home/user/web-servers.nsjail.cfg
+COPY cgi-bin.nsjail.cfg /home/user/cgi-bin.nsjail.cfg
+
+VOLUME /var/log/apache2
```
Comment (sroettger) [inline dist/challenge-templates/web/challenge/Dockerfile:67]: this will add a tmpfs?

**Comment by sroettger** [comment]
cool!

**Review by sroettger — APPROVED** [review]

File: dist/challenge-templates/web/challenge/Dockerfile (hunk 1 above)
Comment (sirdarckcat) [reply dist/challenge-templates/web/challenge/Dockerfile:67]: it's a HD-backed file I think. https://docs.docker.com/engine/reference/builder/#volume
We can make it a tmpfs instead.

Comment (sirdarckcat) [reply dist/challenge-templates/web/challenge/Dockerfile:67]: I'll merge now, but happy to tmpfs it l8r.

**Pull request merged**